src/lib/utils.js

```diff
@@ -138,6 +140,24 @@ const createActZip = async (zipName) => {
     await archive.finalize();
 };
 
+/**
+ * Get act imput from local store
+ * @return {{body: *, contentType: string}}
+ */
+const getLocalInput = () => {
+    const defaultLocalStorePath = path.join(process.cwd(), DEFAULT_LOCAL_EMULATION_DIR,
+        LOCAL_EMULATION_SUBDIRS.keyValueStores, DEFAULT_LOCAL_STORES_ID);
+    const files = fs.readdirSync(defaultLocalStorePath);
+    const inputFileName = files.find(file => !!file.match(/INPUT\..*/));
```

Comment:
This constant "INPUT" should be in constants...

Reply:
Yep, we have that constant in `apify` as `KEY_VALUE_STORE_KEYS`. We should move that to shared.